pom.xml

```diff
@@ -116,24 +128,6 @@
                     <argLine>-XX:MaxPermSize=128m</argLine>
```

Comment:
Could probably be deleted, assuming we are running builds using JDK 8 going forward.